src/main/java/com/jcabi/github/User.java

```diff
@@ -244,6 +275,58 @@ public UserEmails emails() {
             return this.user.emails();
         }
 
+        /**
+         * Returns this user's notifications.
+         * @todo #913:30min Implement the call "List your notifications" as
```

Comment:
Is this todo required still? I think you implemented this method, unless something is still missing here.